src/eos-crash-metrics.c

```diff
@@ -345,6 +350,12 @@ main (int argc, char **argv)
         }
     }
 
+  if (uname(&name) == -1)
```

Comment:
```suggestion
  if (uname (&name) < 0)
```
(sidesteps the problem of “but what if it somehow returns something `<-1`??”)